src/main/java/mate/academy/springboot/swagger/util/SortingUtil.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SortingUtil {
```

Comment:
don't create a bean from the util class; change the class name and package or remove the `Component` annotation and make `sortBy` method static